package.json

```diff
@@ -216,22 +216,34 @@
         "category": "SAS"
       }
     ],
+    "keybindings": [
+      {
+        "when": "editorLangId == sas && !SAS.hideRunMenuItem",
+        "command": "SAS.runSelected",
+        "key": "f3"
+      },
+      {
+        "when": "editorFocus && editorLangId == sas && !SAS.hideRunMenuItem",
+        "command": "SAS.run",
+        "key": "f8"
+      }
+    ],
     "menus": {
       "editor/title/run": [
         {
           "when": "editorFocus && editorLangId == sas && !SAS.hideRunMenuItem",
-          "command": "SAS.run"
+          "command": "SAS.runSelected"
         }
       ],
       "editor/context": [
         {
-          "when": "editorFocus && editorLangId == sas && !SAS.hideRunMenuItem",
-          "command": "SAS.run",
+          "when": "editorLangId == sas && !SAS.hideRunMenuItem",
+          "command": "SAS.runSelected",
```

Comment:
I assumed the order here would define the order these commands appear in the context menu, however Run is still appearing before Run Selected when I run in the debugger. Anyone know how to change this order?

Reply:
The context menu entries are ordered alphabetically. So 'Run SAS Code' is always in front of 'Run Selected SAS Code'

Reply:
https://code.visualstudio.com/api/references/contribution-points#Sorting-inside-groups

Reply:
Great, thanks @scnwwu! I confirmed adding the order attribute to the group did the trick. I'll submit another commit.

Reply:
Another thought is that perhaps we don't need the run entry now, as runSelected(OrAll) should work in most cases. Context menu is just a short cut for frequent actions. Keeping context menu as shorter as possible might be a good idea. Command Palette is the common place for commands.

Reply:
Good point. I'm going to leave the Run command on the context menu for now though, since @crlewi and @skyeaussie reviewed the UI/UX and were good with it. It potentially helps with discovery as well. Last but not least, there are times I have a selection, but my intent is really to run the whole program, so I could see some users preferring to frequently run all the code, ignoring any potential selection.